index.js

```diff
@@ -409,7 +404,7 @@ class ExecutorQueue extends Executor {
             // Note: arguments to enqueue are [queue name, job name, array of args]
             enq = await this.queueBreaker.runCommand('enqueue', this.buildQueue, 'start', [{
                 buildId,
-                jobId,
+                jobId: job.id,
```

Comment:
is this argument supposed to be job name?

Reply:
Nope, jobID https://github.com/screwdriver-cd/models/pull/328/files#diff-26b681bd4dac10f25d5293c7595c435bL430

Reply:
oh okay
probably just need to update the comment on line 404 to say `job id` instead of `job name`

Reply:
i think 404 is meant for the enqueue action. `job name` is `start` in this case